data/User.go

```diff
@@ -0,0 +1,22 @@
+package data
+
+type User struct {
```

Comment:
The new gorm models are currently defined in data package. Open to moving them in respective files where previous models existed.

Reply:
could we use embedded types so we don't have data specific things like "gorm" in the domain layer?